PipelinesTasks/PRMetrics/buildAndReleaseTask/package-lock.json

```diff
@@ -1,5993 +1,8 @@
 {
   "name": "prmetrics",
   "version": "1.1.0",
-  "lockfileVersion": 2,
```

Comment:
Why is this downgraded? :(

Reply:
Looks like my npm version is lower than yours. lockfileVersion is not incremental, it depends on the npm version. Let's keep it as 1 for backwards compatibility.

Reply:
I'd favour using the newer version. I don't think using an old version is a good plan